index.html

```diff
@@ -1248,10 +1208,9 @@ <h2>Terms of Use</h2>
         <p>Terms of use can be utilized by an <a>issuer</a>, <a>subject</a> or a
         <a>holder</a> to express limitations on the use of information
         expressed by the Verifiable Credentials Data Model. The <a>issuer</a> places
-        the terms of use inside the <a>credential</a> before it is converted into a
-        <a>verifiable credential</a>. The <a>holder</a> places the terms of use inside
-        the <a>presentation</a> before it is converted into a
-        <a>verifiable presentation</a>.</p>
+        their terms of use inside the <a>credential</a> before it is converted into a
+        <a>verifiable credential</a>. The <a>holder</a> places issuer terms
```

Comment:
This is not how the data model is designed. The presentation is something created by the holder. The holder can put  credentials (which may or may not contain **issuer termsOfUse**) into the presentation in the `verifiableCredential` property. The holder may also place **holder termsOfUse** into the presentation.

Reply:
Okay. I will look for a way to revise. Where do subject termsOfUse go?

Reply:
For now, I am removing the idea of a holder adding issuer terms of use to a presentation. However, I have logged #224 to track a conceptual mismatch that needs to be resolved.

Reply:
@dhh1128 I think you have misunderstood the current model. The holder is not adding issuer terms of use to a presentation, but rather his/her own terms of use to the presentation. The issuer terms of use are visible in the credential.
Now I do understand your problem with anonymous credentials, in that the VC is opaque. But I would argue that the way to resolve this is not to alter the structure of the VP, but rather, when the holder asserts what the opaque VC contains, that the issuer terms of use are also asserted as well. I suspect this is a protocol issue rather than a data model issue. I realise that a holder may wish to hide certain issuer terms of use, but again, this is a problem that anonymous credentials has to resolve (in its protocol), rather than in the VC data model.